routes/createGames.js

```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var router = express.Router();
+
+const {
+ postGame
+} = require('../controllers/games.controller');
+
+router.get('/compose/new', (req, res) => {
```

Comment:
Nice! What's your thinking on going with this route naming convention?

Reply:
I looked around at other big websites naming for submitting a post or email and decided to go with it.

Reply:
Nice work checking out other sites. I think for our use case this naming convention doesn't create the cohesion we want it to.

The general standard is:
```
/games/:id
/games
/games/new
/games/:id/edit
/games/:id/delete

```